src/com/jalasoft/search_files/utils/validation.py

```diff
@@ -1,35 +1,56 @@
 import os
-
+from src.com.jalasoft.search_files.utils.logging_config import logger
 
 class Validation(object):
     def __init__(self):
         pass
 
     def is_number(self, cadena):
-        response_bool = cadena.isdigit()
-        return response_bool
+        logger.info("Starting the method")
+        boolean_contain = cadena.isdigit()
+        logger.info("The value return is: %s", boolean_contain)
```

Comment:
please change the meesage, it could be:
logger.info("The value returned is: %s", boolean_contain)